From b6955e496179f113d947168c45b63ea9e3c01eda Mon Sep 17 00:00:00 2001
From: Dmitry Lavrenov <lawrenowdima@gmail.com>
Date: Wed, 27 Nov 2024 19:33:45 +0300
Subject: [PATCH] Derive Default

---
 src/lib.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib.rs b/src/lib.rs
index 7a844db..9ebe660 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,6 +1,7 @@
 //! Batumi lib example.
 
 /// Batumi.
+#[derive(Default)]
 pub struct Batumi;
 
 /// Version.
-- 
2.39.2 (Apple Git-143)

